tests/src/Kernel/OgDeleteOrphansTest.php

```diff
@@ -142,14 +151,19 @@ public function testDeleteOrphans($plugin_id, $run_cron, $asynchronous, $queue_i
       $this->container->get('cron')->run();
     }
 
+    // Run shutdown functions if needed.
+    if ($run_shutdown_functions) {
+      _drupal_shutdown_function();
+    }
+
     // Simulate the initiation of the queue process by an asynchronous operation
     // (such as pressing the button that starts a batch operation).
     if ($asynchronous) {
       $this->process($queue_id, $plugin_id);
     }
 
     // Verify the group content is deleted.
-    $this->assertFalse($this->group_content, 'The orphaned node is deleted.');
+    $this->assertNull(Node::load($this->groupContent->id()), 'The orphaned node is deleted.');
```

Comment:
OK this makes a lot of sense, to actually check that the entity is no longer available in the entity storage.